tests/workflow_contracts/workflow_policy_helpers_test.py

```
@@ -146,6 +146,86 @@ def test_it_reads_each_supported_shape(
         """Reading a shape as no labels would exempt the job from placement."""
         assert _job({"runs-on": declared}).runner_labels == expected
 
+    @pytest.mark.parametrize(
+        ("declared", "expected"),
+        [
+            (
+                "${{ github.event_name == 'schedule' && 'ubuntu-latest' "
+                "|| 'ubicloud-standard-8' }}",
```

Ruff (0.16.3)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)